plugins/coreboot/fu-plugin-coreboot.c

```diff
@@ -92,6 +93,13 @@ fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 	fu_device_set_vendor (dev, fu_plugin_get_dmi_value (plugin, FU_HWIDS_KEY_MANUFACTURER));
 	fu_device_add_instance_id (dev, "main-system-firmware");
 
+	/* set vendor ID as the BIOS vendor */
+	dmi_vendor = fu_plugin_get_dmi_value (plugin, FU_HWIDS_KEY_BIOS_VENDOR);
```

Comment:
Isn't this hardcoded to "coreboot"`  (see `fu_plugin_startup`)?

Reply:
Ahh, good catch. Thanks.